sdk/src/Services/DynamoDBv2/Custom/Conversion/DynamoDBEntryConversion.cs

```diff
@@ -651,19 +696,6 @@ public virtual bool TryFrom(Document d, Type targetType, out object result)
 
     internal abstract class Converter<T> : Converter
     {
-        public override IEnumerable<Type> GetTargetTypes()

```

Comment:
I removed this override in the abstract class to instead have the final subclasses return their target types without using reflection.